### DISCUSSION TITLE
OptionMenu triangle color

### DISCUSSION THREAD

**Discussion by fenneltheloon** [TomSchimansky/CustomTkinter#370] [Q&A]
Hi,

Wondering if there is a way to customize specifically the color triangle on the side of an OptionMenu without changing the text color of the left side of the box. Currently I have a box that looks like this:
<img width="146" alt="image" src="https://user-images.githubusercontent.com/22987660/183327000-369f94d7-af4e-4059-8802-be2b476271df.png">
and I'd like the triangle to be white. Scrolling through the docs and source code, there didn't seem to be an obvious way to accomplish this.

**Comment by abhaymakes** [comment]
As far as i can tell, there is no way to change the triangle color individually, but it could be a good feature to have.
